### PR TITLE
Add QPen method in QPainter class

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

File: crates/cxx-qt-lib/src/gui/qpainter.rs [hunk 1]
```diff
@@ -158,6 +163,10 @@ mod ffi {
         #[rust_name = "set_opacity"]
         fn setOpacity(self: Pin<&mut QPainter>, opacity: f64);
 
+        /// Draws the outline (strokes) the path path with the pen specified by pen
+        #[rust_name = "stroke_path"]
+        fn strokePath(self: Pin<&mut QPainter>, path: &QPainterPath, pen: &QPen);
```
Comment (ahayzen-kdab) [inline crates/cxx-qt-lib/src/gui/qpainter.rs:168]: try to keep the ordering A->Z as it becomes increasingly hard to find methods :-)

File: crates/cxx-qt-lib/src/gui/qpainter.rs [hunk 2]
```diff
@@ -123,6 +125,9 @@ mod ffi {
         /// Returns the opacity of the painter. The default value is 1.
         fn opacity(self: &QPainter) -> f64;
 
+        /// Returns the painter's current pen.
+        fn pen(self: &QPainter) -> &QPen;
```
Comment (ahayzen-kdab) [inline crates/cxx-qt-lib/src/gui/qpainter.rs:129]: hmm what about `setPen(const QPen&)` we need some way of solving / naming the overloads here :thinking: Or whether we always go to the most basic like QColor in this case (i think we have done this elsewhere)

**Review by LeonMatthesKDAB — APPROVED** [review]
LGTM

**Comment by ahayzen-kdab** [comment]
Looks good, we need to figure out a plan for overloads at some point...

**Pull request merged**